include/triton/Dialect/Triton/IR/TritonOps.td

```diff
@@ -1133,7 +1134,7 @@ def TT_ExperimentalDescriptorLoadOp : TT_Op<"experimental_descriptor_load", [
 }
 
 def TT_ExperimentalDescriptorStoreOp : TT_Op<"experimental_descriptor_store", [
-  MemoryEffects<[MemWrite<GlobalMemory>]>]> {
+  MemoryEffects<[MemRead<GlobalMemory>, MemWrite<GlobalMemory>]>]> {
```

Comment:
Drive-by change. We read the TMA descriptor from global memory so this has a read effect.